docs/release-notes/highlights-2.13.0.asciidoc

```diff
@@ -1,6 +1,11 @@
 [[release-highlights-2.13.0]]
 == 2.13.0 release highlights
 
+[float]
+[id="{p}-2130-known-issues"]
+=== Known issues
+Upgrading Elasticsearch to version `8.14.0` from any version prior to `8.4.0` leads to Elasticsearch Pods not reaching a ready state. The workaround is to first update to `8.13.2` and then upgrade to `8.14.0`.
```

Comment:
```suggestion
Upgrading Elasticsearch to version `8.14.0` from any version prior to `8.4.0` leads to Elasticsearch Pods not reaching a ready state. To avoid the issue, first upgrade to `8.13.2` and then upgrade to `8.14.0`.
```

Reply:
Technically, an upgrade to any version `8.4.0<= v < 8.14.0` would work followed by the 8.14 upgrade, correct?

Reply:
That is true, I can adjust the message

Reply:
I updated the message, let me know what you think.